bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -252,19 +250,20 @@ private func dummy() {
     _ = NSLocalizedString("Select a color", comment: "")
     _ = NSLocalizedString("Select a color above", comment: "")
     _ = NSLocalizedString("Select a layout", comment: "")
-    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select font size option value e.g: \"Selected: Large\".\ntranslators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: Default", comment: "")
     _ = NSLocalizedString("Set as Featured Image ", comment: "")
     _ = NSLocalizedString("Show post content", comment: "")
     _ = NSLocalizedString("Slash inserter results", comment: "translators: Slash inserter autocomplete results")
     _ = NSLocalizedString("Some blocks have additional settings. Tap the settings icon on the bottom right of the block to view more options.", comment: "")
-    _ = NSLocalizedString("Sorting and filtering", comment: "")
     _ = NSLocalizedString("Start writing…", comment: "")
     _ = NSLocalizedString("Take a Photo", comment: "")
     _ = NSLocalizedString("Take a Photo or Video", comment: "")
     _ = NSLocalizedString("Take a Video", comment: "")
     _ = NSLocalizedString("Tap here to show help", comment: "")
     _ = NSLocalizedString("Tap to hide the keyboard", comment: "")
     _ = NSLocalizedString("Text formatting controls are located within the toolbar positioned above the keyboard while editing a text block", comment: "")
+    _ = NSLocalizedString("The basics", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/editor-help/index.native.js#L134:L134)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.